Geta.EPi.Extensions/UrlExtensions.cs

```diff
@@ -75,5 +75,45 @@ private static Uri GetBaseUri()
                 ? SiteDefinition.Current.SiteUrl
                 : new Uri(HttpContext.Current.Request.Url.GetLeftPart(UriPartial.Authority));
         }
+
+        /// <summary>
+        ///     Gets the Url type of provided Url
+        /// </summary>
+        /// <param name="url"></param>
+        /// <returns>The type of Url (Internal, External). Returns Unknown if resolution fails.</returns>
+        public static UrlType GetUrlType(this Url url)
+        {
+            if (url.IsAbsoluteUri)
```

Comment:
Does not seems to be right. I can have absolute URL that points to internal content. Or am I missing something?